upi/vsphere/stage2/9.post-deployment/playbooks/infra.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- hosts: localhost 
+  tasks:
+  - name: Create default nodeSelector to target workers
+    k8s:
+      state: present
+      definition:
+        apiVersion: config.openshift.io/v1
+        kind: Scheduler
+        metadata:
+          name: cluster
+        spec:
+          defaultNodeSelector: node-role.kubernetes.io/worker=""
+          mastersSchedulable: false
+        policy:
+          name: ""
+
+  - name: Get nodes
+    k8s_facts:
+      kind: Node
+    register: nodes
+
+  - name: Build nodeLabelRaw dicts
+    set_fact:
+      infraNodes: '{{ infraNodes | default([]) + [ nodes | to_json | from_json | json_query(infraNodesQuery) ] }}'
+    vars:
+      infraNodesQuery: "resources[?contains(metadata.name,'infra-')].metadata.name"
+
+  - name: Add infra node-role and remove worker node-role from infra nodes
+    k8s:
+      state: present
+      definition:
+        apiVersion: v1
+        kind: Node
+        metadata:
+          name: "{{ item }}"
+          labels:
+            node-role.kubernetes.io/infra: ""
+            node-role.kubernetes.io/worker:
+    with_items: "{{ infraNodes }}"
+
+  - name: Schedule default ingresscontroller pods to infra nodes
+    k8s:
+      state: present
+      definition:
+        apiVersion: operator.openshift.io/v1
+        kind: IngressController
+        metadata:
+          name: default
+          namespace: openshift-ingress-operator
+        spec:
+          nodePlacement:
+            nodeSelector:
+              matchLabels:
+                node-role.kubernetes.io/infra: ""
```

Comment:
This is literally setting the default node selector to two double-quotes, so nothing wants to schedule because it is looking for 

node-role.kubernetes.io/infra: \"\"

So pods don't schedule with errors like:

"Error creating build pod: unable to parse requirement: label value: Invalid value: "\"\"": must match format [ DNS 1123 subdomain / ] DNS 1123 label"